lib/testScenario/reportGenerator.ts

```diff
@@ -420,9 +420,8 @@ export class ReportGenerator {
         })
         .filter((it) => it !== undefined) as ResponseDiffItem[];
       return delta;
-    } catch (err) {
-      console.log(err);
```

Comment:
Do you mean log will cause exception?

Reply:
Yes. Do not output error message here. It make confuse for customer